index.html

```diff
@@ -4939,18 +4939,56 @@ <h3>Unsigned Claims</h3>
       </section>
 
       <section class="informative">
-        <h3>Token Binding</h3>
+        <h3>Man-in-the-Middle (MITM), Replay, and Cloning Attacks</h3>
+
+        <p>
+The data model does not inherently prevent
+<a href="https://en.wikipedia.org/wiki/Man-in-the-middle_attack">Man-in-the-Middle (MITM)</a>,
+<a href="https://en.wikipedia.org/wiki/Replay_attack">replay</a>, and 
+<a href="https://en.wikipedia.org/wiki/Spoofing_attack">spoofing</a> attacks.
+Both online and offline use cases may be susceptible to these types of attacks,
+where an adversary intercepts, modifies, re-uses, and/or replicates the verifiable
+credential data during transmission or storage. 
+        </p>
+        <h4>Man-in-the-Middle (MITM) Attack</h4>
 
         <p>
 A <a>verifier</a> might need to ensure it is the intended recipient of a
 <a>verifiable presentation</a> and not the target of a
 <a href="https://en.wikipedia.org/wiki/Man-in-the-middle_attack">man-in-the-middle attack</a>.
-Approaches such as token binding [[RFC8471]], which ties the request for a
-<a>verifiable presentation</a> to the response, can secure the protocol.
+Some <a href=#securing-verifiable-credentials>securing mechanisms</a>, like [[VC-JOSE-COSE]],
+provide an option to specify the intended audience of a presentation, which can
+help reduce this risk.
```

Comment:
```suggestion
Some <a href=#securing-verifiable-credentials>securing mechanisms</a>, 
like [[VC-JOSE-COSE]] or [[VC-DATA-INTEGRITY]], provide an option to specify 
the intended audience or domain of a presentation, which can help reduce 
this risk.
```

Reply:
See `domain`, `challenge`, and `nonce` here:

https://w3c.github.io/vc-data-integrity/#proofs

and how those properties are used in VC query languages and protocols here:

https://w3c-ccg.github.io/vp-request-spec/#did-authentication